feat: added packet count telemetry metrics (#2797)
interledger/rafiki: packages/documentation/src/content/docs/telemetry/overview.md

The objective of the telemetry feature is to gather metrics and establish an infrastructure for visualizing valuable network insights. The metrics we at the Interledger Foundation collect include:

- The total amount of money transferred via packet data within a specified time frame (daily, weekly, monthly).
- The number of transactions that have been at least partially successful.
- The number of ILP packets flowing through the network.
- The average amount of money held within the network per transaction.

> - The avergae time it takes for an outgoing payment to complete.

Reviewer: ```suggestion
- The average time it takes for an outgoing payment to complete
```

## Instrumentation

> Rafiki currently has seven counter metrics. All data points (counter increases and histogram records) are exported to collection endpoints at a configurable interval (default recommended to 15s).

Reviewer 2: This is kinda prone to drift from the truth (count, type of metrics). Feel like we could make our lives easier and avoid potentially misleading people by removing some of the incidental details and rephrasing like:

```suggestion
Data points for all metrics (e.g. counter increases) are exported to collection endpoints at a configurable interval (default recommended to 15s).
```

Currently there are more than 7 metrics (my bad - forgot to include the latest one I added 😬 ) and they arent all counters (`ilp_pay_time_ms` is a histogram for example).
